services/spec_and_run_subscriber.go

```diff
@@ -134,9 +134,10 @@ func defaultParamsFor(task string, log types.Log) models.JSON {
 	switch strings.ToLower(task) {
 	case "ethtx":
 		js := fmt.Sprintf(
-			`{"address":"%s", "functionSelector":"%s"}`,
+			`{"address":"%s", "functionSelector":"%s", "dataPrefix":"%s"}`,
 			log.Address.String(),
 			OracleFulfillmentFunctionID,
+			log.Topics[SpecAndRunTopicInternalID].String(),
```

Comment:
I believe `%s` implicitly calls `.String()` ?

Reply:
I thought you were right, but when I tried removing it, the tests broke. I'm going to leave it as is for this PR.

Reply:
Weird, thanks for checking it out!